sys/shell/commands/sc_gnrc_netif.c

```diff
@@ -144,7 +149,14 @@ static void _set_usage(char *cmd_name)
          "       * \"addr\" - sets (short) address\n"
          "       * \"addr_long\" - sets long address\n"
          "       * \"addr_short\" - alias for \"addr\"\n"
+         "       * \"appeui\" - sets Application EUI\n"
+         "       * \"appkey\" - sets Application key\n"
+         "       * \"appskey\" - sets Application session key\n"
          "       * \"cca_threshold\" - set ED threshold during CCA in dBm\n"
+         "       * \"deveui\" - sets Device EUI\n"
+         "       * \"dr\" - sets datarate\n"
+         "       * \"rx2_dr\" - sets datarate of RX2 (lorawan)\n"
+         "       * \"nwkskey\" - sets Network Session Key\n"
```

Comment:
Those and the other additions in this file should be hidden behind `#ifdef MODULE_GNRC_NETIF_CMD_LORA` so they don't fill up the ROM on devices that don't use LoRA.

Reply:
addressed in 0696f18